File: python/array-of-objects/exercise01/main.py
from classCar import Car

# Instances of the Car class
car1 = Car("Toyota", "Corolla", "Red")
car2 = Car("Honda", "Civic", "Blue")
car3 = Car("Ford", "Mustang", "Black")

# Create array of objects
cars = [car1, car2, car3]

# Create for loop to iterate through the array of cars
for car in cars:
  car.drive()

File: javascript/array-of-objects/exercise01/main.js
const Car = require('./classCar.js');

// Instances
const car1 = new Car("Ford", "Mustang", "Red");
const car2 = new Car("Toyota", "Corolla", "Blue");
const car3 = new Car("Ford", "Fiesta", "White");

// Create array of objects
const cars = [car1, car2, car3];

for (let i = 0; i < cars.length; i++) {
  cars[i].drive();
}